app/src/main/res/layout/layout_editor_bottom_sheet.xml

```diff
@@ -95,7 +99,10 @@
     android:layout_height="wrap_content"
     android:layout_above="@id/space_bottom"
     android:layout_alignParentEnd="true"
-    android:layout_margin="16dp"
+    android:layout_marginStart="16dp"
+    android:layout_marginTop="16dp"
+    android:layout_marginEnd="16dp"
+    android:layout_marginBottom="16dp"
     android:src="@drawable/ic_close" />

```

Comment:
This is effectively the same, right?

Reply:
Yes. Not sure why I have changed it :sweat_smile: